Ultz.SilkExtensions.ImGui/ImGuiController.cs

```diff
@@ -49,7 +49,16 @@ public ImGuiController(GL gl, IView view, IInputContext input)
             IntPtr context = ImGuiNET.ImGui.CreateContext();
             ImGuiNET.ImGui.SetCurrentContext(context);
             var io = ImGuiNET.ImGui.GetIO();
-            io.Fonts.AddFontDefault();
+
+            if (imGuiFontConfig == null)
+            {
+                io.Fonts.AddFontDefault();
+            }
+            else
+            {
+                io.Fonts.AddFontFromFileTTF(imGuiFontConfig.FontPath, imGuiFontConfig.FontSize);
```

Comment:
Did we want to allow the font kind to be specified (by an enum perhaps) in the `ImGuiFontConfig` struct? This probably allows more freedom than just assuming it's a TTF.

Reply:
Currently I will leave it as is. mostly because, from my research, ImGui does not use any other font types by default. I think in next few days, I'll research how to load custom fonts in ImGui, and I guess after that I'll separate it by just being able to pass in ImFontConfig directly to ImGuiFontConfig for more flexible font loading options.

Reply:
All good!

Reply:
Okay, just to clarify. Did some more digging and found out that this function `AddFontFromFileTTF`, also loads `.otf` fonts as well.

Got this info from older ImGui version's [README](https://skia.googlesource.com/external/github.com/ocornut/imgui/+/v1.53/extra_fonts/README.txt) file for extra_fonts section, there it says: 

> (Note: .OTF support in stb_truetype.h currently doesn't appear to load every font)

Then I went ahead and tested with few `.otf` files and it worked without any problems so apparently current functionality supports both `.ttf` and `.otf` fonts.

Reply:
Ah cool, no need for more APIs then!

Reply:
Well then. I guess other stuff is ready! If something needs changing, let me know, and thanks for your time!